v1.5/getting-started/kubernetes/installation/hosted/kubeadm/calico.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 data:
   # The location of your etcd cluster.  This uses the Service clusterIP
   # defined below.
-  etcd_endpoints: "http://100.78.232.136:6666"
+  etcd_endpoints: "http://10.96.232.136:6666"
```

Comment:
Kubeadm was released pretty recently, and our docs for kubeadm were released after that, (around the time we released 1.6), so I think we should change it since more users would be trying an older _stable_ version with kubeadm.